cypress/support/fragments/circulation-log/searchPane.js

```diff
@@ -40,22 +52,35 @@ export default {
     const dateRegEx = /\d{1,2}\/\d{1,2}\/\d{4},\s\d{1,2}:\d{2}\s\w{2}/gm;
 
     return cy.get('#circulation-log-list').then(element => {
```

Comment:
I believe  it be implemented through interactor

Reply:
Unfortunatly nope, because this is container and we don't have interactor for this element. And we don't need interactor for this element because it seems not usual case

Reply:
let's add todo related with redesign to interactors and try to make this later.